deployments/cloudfoundry/buildpack/README.md

```diff
@@ -0,0 +1,168 @@
+# OpenTelemetry Collector Pivotal Cloud Foundry (PCF) Buildpack
+
+A [Cloud Foundry buildpack](https://docs.pivotal.io/application-service/2-11/buildpacks/) to install
+the OpenTelemetry Collector for use with PCF apps.
```

Comment:
```suggestion
the Splunk OpenTelemetry Collector for use with PCF apps.
```
I think we should append `Splunk` for all other references to `OpenTelemetry Collector`.

Reply:
Updated, all references should now be `Splunk OpenTelemetry Collector`.